### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by taz-mon**
Incorporated review comments and finalized PKS and GKE content changes

### CODE REVIEW

File: docs/Deploying-Pega-on-GKE.md [hunk 1]
```diff
@@ -20,197 +20,224 @@ How the deployment works
...
+6. Scroll to the bottom of the page and click **Create**.
+
 ### Create a database resource
 
-Create a PostgreSQL database in which to install Pega Platform. GKE deployments require you to install in an SQL database. When you are finished, you will need to obtain the database name and the server which you are creating in this procedure so you can customize your “pega” Helm chart with this information.
+GKE deployments require you to install Pega Platform software in an SQL database. After you create an SQL instance that is available to your GKE cluster, you must create a PostSQL database in which to install Pega Platform. When you are finished, you will need the database name and the SQL instance IP address which you create in this procedure in order to add this information to your “pega” Helm chart.
+
+#### Create an SQL instance
+
+To begin, create an SQL instance that is available to your GKE cluster. In this example, we create an SQL instance in GCP; however, you can create or use an database resource that is available to the GKE cluster.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
To begin, create an SQL instance that is available to your GKE cluster. In this example, we create an SQL instance in GCP; however, you can create or use any database resource that is available to the GKE cluster.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 2]
```diff
@@ -20,197 +20,224 @@ How the deployment works
...
-    address to whitelist for this postgres database, and click **Done**.
+    d. In **Database version**, select **PostgreSQL 11**.
 
-    For clusters that are provisioned by Pivotal: you can launch the Kubernetes
-dashboard to view the external IP address of the nodes in your cluster; to add
-that IP network to the database whitelist, enter the first three sets of number,
-and use 0/24 for the final set in this IP range. For example: 123.123.123.0/24.
+    e. In **Configuration options \> Connectivity**, select **Public IP**, click **+ Add Network**, enter a **Name** and **Network** of one or more IP address to whitelist for this PostgreSQL database, and click **Done**.
+    
+    As a best practice, you should add the following IP addresses: your local system from where you install helm, the worker nodes of the cluster. One method for finding the IP address for worker nodes of the cluster is to use the command `kubectl describe nodes | grep ExternalIP`.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
    As a best practice, you should add the following IP addresses: your local system from where you install helm, and the worker nodes of the cluster. One method for finding the IP address for worker nodes of the cluster is to use the command `kubectl describe nodes | grep ExternalIP`.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 3]
```diff
@@ -20,197 +20,224 @@ How the deployment works
...
 
     You can see a deployment progress page displayed until your database server
 deployment is complete, which can take up to 5 minutes. When complete, the GCP UI
 displays all of the SQL resources in your account, which includes your newly created PostgreSQL database:
 
 ![cid:image007.png\@01D5A3B1.62812F70](media/9aa072ea703232c2f6651fe95854e8dc.62812f70)
 
+#### Create a database in your SQL instance
+
+You must create a PostSQL database in your new SQL instance into which you will install Pega Platform. Use the database editing tool of your choice to log into your SQL instance and create this new PostgreSQL database. The following example was completed using pgAdmin4.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
You must create a PostgreSQL database in your new SQL instance into which you will install Pega Platform. Use the database editing tool of your choice to log into your SQL instance and create this new PostgreSQL database. The following example was completed using pgAdmin4.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 4]
```diff
@@ -20,197 +20,224 @@ How the deployment works
...
+
+1. Use a database editor tool, such as pgadmin4, to log into your SQL instance.
+
+  You can find your access information and login credentials, by selecting the SQL instance in the GCP console.
+
+2. In the database editor tool, navigate to Databases and create a new database.
+
+  Pega does not require any additional configuration.
+
+With your SQL service IP address and your new database name, you are ready to continute to the next section.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
With your SQL service IP address and your new database name, you are ready to continue to the next section.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 5]
```diff
@@ -242,156 +269,212 @@ To finalize these details, follow these steps:
...
+
+```yaml
+$ kubectl get nodes
+NAME                                             STATUS   ROLES    AGE    VERSION
+gke-taz-gke-runbook-default-pool-7fd38759-2dzk   Ready    <none>   3d2h   v1.13.11-gke.14
+gke-taz-gke-runbook-default-pool-7fd38759-qq4h   Ready    <none>   3d2h   v1.13.11-gke.14
+```
 
-4. To use the gke CLI to download the cluster Kubeconfig access credential file, which is specific to your cluster, into your \<local filepath\>/.kube directory, enter:
+6. Install the Kubernetes dashobard by applying an available version on Github.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
6. Install the Kubernetes dashboard by applying an available version on Github.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 6]
```diff
@@ -242,156 +269,212 @@ To finalize these details, follow these steps:
...
-
-    In order to continue using the Kubernetes dashboard to see the progress of your deployment, keep this PowerShell or Linux command prompt open and open a new one for the remaining steps.
-
-10. Do one of the following:
+    You can now view your deployment details visually using the Kubernetes dashboard. You will use this dashboard to review the status of your deployment as you continue. At this point, with no deployment, you only see the GKE resources. Note that the Kubernetes dashboard does not display your GKE cluster name or your resource name. This is expected behavior.
 
-- Open a new Windows PowerShell running as Administrator on your local system and change the location to the top folder of your gke-demo folder.
+    In order to continue using the Kubernetes dashboard to see the progress of your deployment, keep this Linux shell prompt open and open a new one for the remaining steps.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
    In order to continue using the Kubernetes dashboard to see the progress of your deployment, keep this Linux shell open and open a new one for the remaining steps.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 7]
```diff
@@ -242,156 +269,212 @@ To finalize these details, follow these steps:
...
-$ kubectl create namespace mypega
-namespace/mypega created
+$ kubectl create namespace mypega-gke-demo
+namespace/mypega-gke-demo created
 $ kubectl create namespace pegaaddons
 namespace/pegaaddons created
 ```
 
-12. To install the addons chart, which you already updated during the prepping a local system for your deployment, enter:
+15. Install the addons chart, which you already updated during the prepping a local system for your deployment.
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
15. Install the addons chart, which you already updated when prepping your local system for your deployment.
```

File: docs/Deploying-Pega-on-GKE.md [hunk 8]
```diff
@@ -242,156 +269,212 @@ To finalize these details, follow these steps:
...
+For subsequent Helm installs, use the command `helm install mypega-gke-demo pega/pega --namespace mypega-gke-demo` to deploy Pega Platform and avoid another Pega Platform installation.
 
-A successful Pega deployment immediately returns details that show progress for your deployment.
+A successful Pega deployment immediately returns details that show progress for your `mypega-gke-demo` deployment.
 
-14. Refresh the Kubernetes dashboard you opened in step 7. If you closed the dashboard, open a new command prompt running as Administrator and relaunch the browser as directed in Step 10.
+17. Refresh the Kubernetes dashboard you opened in step 8. If you closed the dashboard, open a new command prompt and relaunch the browser as directed in Step 10.
 
-15. In the dashboard, use the **Namespace** pulldown to change the view to **mypega** and click on the **Pods** view.
+18. In the dashboard, use the **Namespace** pulldown to change the view to `mypega-gke-demo` and click on the **Pods** view. Initiallly, you can some pods have a red status, which means they are initializing:
```
Comment (pega-talba) [inline docs/Deploying-Pega-on-GKE.md]:
```suggestion
18. In the dashboard, use the **Namespace** pulldown to change the view to `mypega-gke-demo` and click on the **Pods** view. Initially, some pods will have a red status, which means they are initializing:
```

File: docs/prepping-local-system-runbook-windows.md [hunk 9]
```diff
@@ -31,17 +31,25 @@ Where \<platform\>-demo is:
...
 You are ready to continue preparing your local system.
 
 Installing required applications for the deployment
 ---------------------------------------------------
 
+The entire deployment requires the following applications to be used a somee point during the process, so it's useful to prepare your local system with all of the files before you start your deployment:
+- Helm
+- kubectl
+- Docker
+- unzip (or something equivalent to extract files from .zip archive files.)
```
Comment (pega-talba) [inline docs/prepping-local-system-runbook-windows.md]: Should we recommend 7zip?

File: docs/prepping-local-system-runbook-windows.md [hunk 10]
```diff
@@ -443,22 +387,24 @@ For details about logging into Docker from a secure password file using the `--p
 
 3. Create a text file with the text editor of your choice in the \<local filepath\>\\\<platform\>-demo\\\<pega-distribution-image\> folder where you extracted the Pega distribution on your local system.
 
-You can list the folder content and see folders for Pega archives, Images, rules, and scripts.
+From this folwder, you can list the folder content and see folders for Pega archives, Images, rules, and scripts.
```
Comment (pega-talba) [inline docs/prepping-local-system-runbook-windows.md]:
```suggestion
From this folder, you can list the folder content and see folders for Pega archives, Images, rules, and scripts.
```

File: docs/prepping-local-system-runbook-windows.md (hunk 9 above)
Comment (taz-mon) [reply docs/prepping-local-system-runbook-windows.md]: for windows, the unzip command is there by default. I didn't want to go specifically into adding a tool when Windows would do it...I think we're covered with (or something equivalent to extract files from .zip archive files.)

**Pull request merged**